### PR TITLE
ci(android): Update release workflow

### PR DESCRIPTION
**Pull request by Achintha444**
## Purpose
Update the release workflow, Github action.

### CODE REVIEW

File: .github/workflows/scripts/android/commit_and_push.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+#!/bin/bash
+
+$GITHUB_RUN_NUMBER=$1
+$RELEASE_BRANCH=$2
+$MASTER_BRANCH="master"
+
+# Go to root directory
+go_to_root_dir() {
+  cd cd ../../../../
```
Comment (JayaShakthi97) [inline .github/workflows/scripts/android/commit_and_push.sh:29]: Is this expected?
```suggestion
  cd ../../../../
```

Comment (Achintha444) [reply .github/workflows/scripts/android/commit_and_push.sh:29]: Ack will fix this

Comment (Achintha444) [reply .github/workflows/scripts/android/commit_and_push.sh]: Fixed with https://github.com/asgardeo/mobile-ui-sdks/pull/2/commits/a8896adcad5c91a9c3e873570d395ddd67bf13fe

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -11,7 +31,6 @@ on:
         default: 'android'
         options:
         - android
-        - android-core
       version:
         type: choice
         description: Choose which version to bump before release
```
[the remark this answers is not in the document]
Comment (pavinduLakshan) [reply .github/workflows/release.yml:36]: For version, shall we add `patch`, `minor` and `major` as the options?

**Review by brionmario — APPROVED** [review]

**Pull request merged**